microsoft/botframework-solutions
# [Skill Template][TypeScript] Add tests for the template

## Description

- Add new tests
  - `mainDialogTest`
  - `sampleDialogTest`
  - `localizationTest`
  - `interruptionTest`
- Update `sample-skill` with tests
- Fix `ESLint` errors
- Update `.eslintignore` file to ignore `skill` folder

Close #1199 

## Testing Steps
1. Go to `AI\templates\Virtual-Assistant-Template\typescript\samples\sample-assistant`
1. Run `npm install` and `npm run build`
1. Run `npm run test`
1. Check that all the test passed successfully

## Checklist
